src/amqp10_msg.erl

```diff
@@ -411,6 +411,10 @@ set_message_annotations(
     Anns1 = #'v1_0.message_annotations'{content = maps:to_list(Anns)},
     Msg#amqp10_msg{message_annotations = Anns1}.
 
+wrap_ap_value(true) ->
```

Comment:
ok so interestingly AMQP 1.0 have to different ways to encode a boolean and we support the both - using the tuple `{boolean, bool()}` will encode the boolean using two bytes - alternatively you can just return `true` or `false` then it will encode the boolean using one byte. Either is fine.

Reply:
Thanks, that was my understanding as well.